packages/nextjs/test/integration/test/client/errorClick.test.ts

```diff
@@ -0,0 +1,17 @@
+import { getMultipleSentryEnvelopeRequests } from './utils/helpers';
+import { test, expect } from '@playwright/test';
+import { Event } from '@sentry/types';
+
+test('should capture error triggered on click', async ({ page }) => {
+  await page.goto('/errorClick');
+
+  const [_, events] = await Promise.all([
+    page.click('button'),
+    getMultipleSentryEnvelopeRequests<Event>(page, 1, { envelopeType: 'event' }),
```

Comment:
Potential race condition

Reply:
We've done this before with no issues: https://github.com/getsentry/sentry-javascript/blob/59e51bd5cc148835adee41e2ca5207a5052606c5/packages/integration-tests/suites/sessions/update-session/test.ts#L11

We can adjust this if it is causing flakes.